module/users/helper.js

```diff
@@ -467,6 +467,62 @@ module.exports = class UsersHelper {
           });
         }
 
+        let totalCount = 0;
+        let mergedData = [];
+        //fetch all targeted programs to users
+        let targetedPrograms = await programsHelper.forUserRoleAndLocation(
```

Comment:
Updated Function to take additional parameter as programId and return id only when that program is targeted else if its non targeted then it will not return program Id